qt/python/mantidqt/widgets/colorbar/colorbar.py

```diff
@@ -258,3 +268,24 @@ def redraw(self):
         self.colorbar.draw_all()
         self.canvas.draw_idle()
         self.colorbarChanged.emit()
+
+    def set_default_scale(self):
+        scale = 'Linear'
+        scale_setting = []
+        if self.conf and self.conf.has("sliceviewer/scale_norm"):
+            scale_setting = self.conf.get("sliceviewer/scale_norm")
```

Comment:
When saving should we use a separate key for the power value to avoid having to do this decoding? Something like `sliceviewer/scale_norm` says 'Power' and then there is an additional key 'sliceviewer/scale_norm_power' stores the value?